src/test/java/org/folio/rest/support/ApiTests.java

```diff
@@ -97,4 +98,11 @@ protected void checkNotFound(URL url)
 
     assertThat(statusCode, is(HttpURLConnection.HTTP_NOT_FOUND));
   }
+
+  protected void doesNotHaveProperty(String property, JsonObject resource, String type) {
```

Comment:
This likely belongs in a separate place to the ApiTests base class, maybe as a custom matcher?

Reply:
This has been reworked.